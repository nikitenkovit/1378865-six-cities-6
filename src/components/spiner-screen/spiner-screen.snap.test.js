import React from "react";
import {render} from "@testing-library/react";
import SpinerScreen from "./spiner-screen";

it(`Should spiner render correctly`, () => {
  const {container} = render(<SpinerScreen/>);
  expect(container).toMatchSnapshot();
});
