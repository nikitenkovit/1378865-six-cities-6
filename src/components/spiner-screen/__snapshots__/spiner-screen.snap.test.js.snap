// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should spiner render correctly 1`] = `
<div>
  <div
    class="spiner"
  >
    <div
      class="loader"
    >
      <div
        class="ball"
      />
      <div
        class="ball"
      />
      <div
        class="ball"
      />
    </div>
    <svg
      version="1.1"
      xmlns="http://www.w3.org/2000/svg"
    >
      <defs>
        <filter
          id="gooey"
        >
          <fegaussianblur
            in="SourceGraphic"
            result="blur"
            stdDeviation="12"
          />
          <fecolormatrix
            in="blur"
            mode="matrix"
            result="goo"
            values="1 0 0 0 0  0 1 0 0 0  0 0 1 0 0  0 0 0 18 -8"
          />
          <feblend
            in="SourceGraphic"
            in2="goo"
          />
        </filter>
      </defs>
    </svg>
  </div>
</div>
`;
